### DISCUSSION TITLE
In the windows version Openradioss the Implicit solver need mumps?

### DISCUSSION THREAD

**Discussion by andprad74** [OpenRadioss/OpenRadioss#1407] [Contributors]
Hello everybody and thanks for the great job.
I started to learn openradioss precompiled version for linux and win.

In the windows version the explicit solver seems to works fine , the implicit needs mumps.
How to solve that issue ?

![implicit_solver_need_mumps](https://user-images.githubusercontent.com/131922029/234715828-c1fdf5ea-7e74-4d78-a317-a17f9b5cf008.png)

**Comment by MarianBulla** [comment]
Hello andprad74,

regaring your question, please see this discussion, where it is answered: https://github.com/orgs/OpenRadioss/discussions/485

Hope it helps you ?

Thanks and best regards,
Marian

**Comment by andprad74** [comment]
Thanks Marian,
I checked the answer, probably It Is to much complicated for my Linux  ability but i Will try. Thanks again .
